src/components/forms/PatientForm.tsx

```diff
@@ -47,9 +41,10 @@ const PatientForm: FunctionComponent<PatientFormProps> = ({ patient, isEditForm
     try {
       const data = await createPatient(token, patientState)
       setPatientState(data)
+      addMessage({ text: 'Patient created successfully', variant: 'success' })
```

Comment:
I would address these in a dedicated PR since we also need to deal with the translation and probably helper method e.g. `flashSuccessMessage()` that deals with default title and variant